wisembly/amqp-bundle/1.3/post-install.txt

```diff
@@ -0,0 +1,8 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * <fg=blue>Add</> "gates" to the bundle configuration
```

Comment:
Is this phrase easy to understand for people installing the bundle? I know I don't understand a word of it (*"Add gates to the bundle config"* ?!?!?!) but maybe it's because I'm not skilled in this AMQP thing.

Reply:
Documented here : https://github.com/Wisembly/AMQPBundle#concept-of-gates

Reply:
But as I added a acme gate example and documented it a bit more in the config file, I think that the post-install is now unnecessary now.

Reply:
Please remove it then. We try to limit display during installation as much as possible as it's not permanent.